transmogrifier/sources/ead.py

```diff
@@ -60,6 +86,35 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
                             identifier=self.generate_name_identifier_url(name_element),
                         )
                     )
+        # dates
```

Comment:
I will likely expand the date parsing code after we have the TIMX-93 discussion about date issues